libnavui-dropin/src/main/java/com/mapbox/navigation/dropin/ViewStyleCustomization.kt

```diff
@@ -171,10 +174,18 @@ class ViewStyleCustomization {
         fun defaultInfoPanelBackground(): Int = R.drawable.mapbox_bg_info_panel
 
         /**
-         * Default destination marker icon.
+         * Default [PointAnnotationOptions] for showing destination marker.
          */
-        @DrawableRes
-        fun defaultDestinationMarker(): Int = R.drawable.mapbox_ic_destination_marker
+        fun defaultMarkerAnnotationOptions(context: Context): PointAnnotationOptions =
```

Comment:
```suggestion
        fun defaultDestinationMarkerAnnotationOptions(context: Context): PointAnnotationOptions =
```

Reply:
I prefer to keep it as is and hence I am not applying the suggested change.